src/applications/mhv-landing-page/containers/AppConfig.jsx

```diff
@@ -32,7 +32,13 @@ const AppConfig = ({
     () => {
       const id = profile?.accountUuid;
       if (id) {
-        setDatadogRumUserFn({ id });
+        const hasEHRM = profile?.facilities?.some(
+          facility => facility.isCerner,
+        );
+        const hasVista = profile?.facilities?.some(
+          facility => !facility.isCerner,
+        );
```

Comment:
We keep our selectors in `src/applications/mhv-landing-page/selectors`. Could you move these selectors there, and add a few lines describing their intent as JSDoc comments? I might be able to infer what `hasEHRM` means, but I'd rather be told what it means.